src/store/modules/exampleModule.js

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+
+    state: {
+        isLoggedIn: false,
+        errorMsg: "login failed",
+        // axiosInstance: null,
+        tokenAuth: false
+    },
+    mutations: {
+        "login": (state) => {
+            state.isLoggedIn = true
+        },
+        "Token/login": (state) => {
+            state.tokenAuth = true
+        }
+    },
+    getters: {
+
+        hasToken(state) { return state.tokenAuth }
+
+    },
+    actions: {
+        login: async (context, payload) => {
+
+            await axios.post('https://claplablmsapi.clap.co.il/auth/login/1',
+                {
+                    email: payload.email,
+                    password: payload.password
+                }
+            ).then((response) => {
+                localStorage.setItem('access_token', response.data.token)
+                context.commit("login");
```

Comment:
send the user's data you're getting from the server to the commit. 
then store the user's data on the state. 
apps usually needs the data of the user